### ISSUE TITLE
New buildout parser does not work with buildout.coredev

### ISSUE THREAD

**Issue by jensens** [plone/plone.versioncheck#36]
While running 1.6.0 with new buildout parser I got the following error:

```
~w/cdev/plone5(5.1*)$ ./bin/versioncheck -pnNb >versions.html 
Parsing buildout files:
- buildout.cfg
- core.cfg
- themepreview.cfg
- versions.cfg
  291 entries in versions section.
- http://dist.plone.org/versions/zope-2-13-26-versions.cfgTraceback (most recent call last):
  File "./bin/versioncheck", line 19, in <module>
    sys.exit(plone.versioncheck.script.run())
  File "/home/jensens/.buildout/shared-eggs/plone.versioncheck-1.6.0-py2.7.egg/plone/versioncheck/script.py", line 123, in run
    pkgsinfo['pkgs'] = parse(args.buildout)
  File "/home/jensens/.buildout/shared-eggs/plone.versioncheck-1.6.0-py2.7.egg/plone/versioncheck/parser.py", line 121, in parse
    relative=base_relative
  File "/home/jensens/.buildout/shared-eggs/plone.versioncheck-1.6.0-py2.7.egg/plone/versioncheck/parser.py", line 107, in _extract_versions_section
    versionannotation_section_name
  File "/home/jensens/.buildout/shared-eggs/plone.versioncheck-1.6.0-py2.7.egg/plone/versioncheck/parser.py", line 107, in _extract_versions_section
    versionannotation_section_name
  File "/home/jensens/.buildout/shared-eggs/plone.versioncheck-1.6.0-py2.7.egg/plone/versioncheck/parser.py", line 107, in _extract_versions_section
    versionannotation_section_name
  File "/home/jensens/.buildout/shared-eggs/plone.versioncheck-1.6.0-py2.7.egg/plone/versioncheck/parser.py", line 45, in _extract_versions_section
    buildout = Buildout(filename, [])  # Use zc.buildout parser
  File "/home/workspacejensens/cdev/plone5/lib/python2.7/site-packages/zc/buildout/buildout.py", line 347, in __init__
    self._buildout_dir = options['directory']
  File "/home/workspacejensens/cdev/plone5/lib/python2.7/site-packages/zc/buildout/buildout.py", line 1370, in __getitem__
    raise MissingOption("Missing option: %s:%s" % (self.name, key))
zc.buildout.buildout.MissingOption: Missing option: buildout:directory
```

**Comment by loechel** [comment]
Ok will care about that

**Comment by jensens** [comment]
Fixed by #38
